functional/cypress-tests/cypress/fixtures/Dashboard.js

```diff
@@ -1,12 +1,13 @@
-export function dashboardNameGen(name = "Dashboard") {
-  return name + Date.now().toString();
+export function dashboardNameGen(name = 'Dashboard') {
+    return name + Date.now()
+        .toString()
 }
 
-export let columnEdgeValues = ["3", "4", "20", "21"];
+export const columnEdgeValues = ['3', '4', '20', '21']
 
-export let switchIntervalEdgeValues = ["2", "3"];
+export const switchIntervalEdgeValues = ['2', '3']
 
-export let dashboardNames = [
-  dashboardNameGen(),
-  "Long Dashboard Name Input. It has exactly 51 chars."
-];
+export const dashboardNames = [
+    dashboardNameGen(),
+    'Long Dashboard Name Input. It has exactly 51 chars.'
```

Comment:
Webapp is using tab size of 2: https://prettier.io/docs/en/options.html#tab-width

Reply:
Aligned